### PR TITLE
[poap-with-weight] Poap with weight , added supply parameter

### PR DESCRIPTION
**Pull request by Rashmi-278**
Fixes # .

Changes proposed in this pull request:
-  gql query created
-  used utils to query

### CODE REVIEW

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Hello, Please revert changes in the `package-lock.json` file, which is failing the build/checks, Thanks

File: src/strategies/poap-with-weight/index.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+// const getTokenSupply = gql`
+//   query($tokenId: Number!) {
+//     token(id: $tokenId) {
+//       event {
+//         tokenCount
+//       }
+//     }
+//   }
+// `;
```
Comment (ChaituVR) [inline src/strategies/poap-with-weight/index.ts:23]:
```suggestion
```

File: src/strategies/poap-with-weight/index.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    provider,
+    abi,
+    options.ids.map((id: any) => [options.address, 'ownerOf', [id]]),
+    { blockTag }
+  );
+  const poapWeights = {};
+  Object.keys(options.id).map((k) => {
+    poapWeights[k] = 1000 / options.id[k];
+  });
+  // return response[0].owner;
```
Comment (ChaituVR) [inline src/strategies/poap-with-weight/index.ts:66]:
```suggestion
```

File: src/strategies/poap-with-weight/index.ts [hunk 3]
```diff
@@ -0,0 +1,77 @@
+import { gql, useQuery } from '@apollo/client';
+import { VariableType } from 'json-to-graphql-query';
+import { multicall , subgraphRequest } from '../../utils';
+
+export const author = 'G2 & Torch';
```
Comment (ChaituVR) [inline src/strategies/poap-with-weight/index.ts:5]: This should be a GitHub Id

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Comment by bonustrack** [comment]
@Rashmi-278 Hi! We've moved the strategies to another repository here:  https://github.com/snapshot-labs/snapshot-strategies Could you make your PR there? Sorry for the trouble!

**Comment by bonustrack** [comment]
Closing this PR for now, new strategies must be added into this repository now: https://github.com/snapshot-labs/snapshot-strategies, feel free to reopen this PR if needed.

**Pull request closed without merging**